cherrypy/_cplogging.py

```diff
@@ -262,6 +262,8 @@ def access(self):
                  'f': dict.get(inheaders, 'Referer', ''),
                  'a': dict.get(inheaders, 'User-Agent', ''),
                  'o': dict.get(inheaders, 'Host', '-'),
+                 'i': str(uuid.uuid4()),
+                 'z': self.time_z(),
```

Comment:
What about exposing these atoms to error log?

Reply:
you are right, I'll check how to do it

Reply:
I read the code and I'm in a little in trouble: in the error_log the atoms aren't used at all, is that right? If it's right I think that adding this support could be out of scope of this PR and I can work on that with a different PR if there's the interest about that

Reply:
well, supporting these atoms we allow users use them if they want to change that error_log format template. Okay, you can work on this separately.